data/avgi/avgi side missions.txt

```diff
@@ -0,0 +1,1067 @@
+			choice
+				`	"The Guard?"`
+				`	"Why is that?"`
+					goto why
+
+			`	Claret flares her wings out and raises her arms in an exaggerated pose. "Inflated boast. The Twilight Guard. You know, us brave warriors that are the last line of defense between the murderous Aberrant and the last bastion of Avgi civilization. Well, except for the people stuck back on Aviskir, but we don't talk about that, heh." She waves an arm. "But back to you, Captain <last>."`
+
+			label why
+			`	She points at you dramatically. "You're an alien, and you fought the Aberrant. Even though we've seen a few of you people around, none of them have flown a starship quite like yours. Of course we've been talking about you. Wouldn't surprise me if half the Consonance was, really."`
+			choice
+				`	"Well, what do they say about me?`
+					goto gossip
+				`	"All good things, I hope."`
+
+			`	"Dismissive. Of course, everyone is singing your praises. Except for the ones who claim you aren't one of those 'humans' at all, and are secretly an Aberrant in the shape of one waiting for the perfect moment to strike and eat our children." Claret gives you a serious look. "You aren't, right?"`
+			choice
+				`	"Not last I checked."`
+				`	"Nope."`
+				`	"Well, you've caught me."`
+					goto caught
+
+			label caught
+				`"Oh no." she says in a completely flat voice. "I am absolutely terrified." She takes a deep drink before sighing contentedly.`
+
+			label gossip
+				`	"Some say that the humans will soon arrive to save us with their superior technology. Others say you'll be the next aliens to invade us, and finish the job the Aberrant started."`
+			choice
+				`	"And which do you believe?"`
+					goto believe
+				`	"Very contradictory rumors."`
+
+			`	She gives you an incredulous look. "Scorn. When are they not? I don't know about you humans, but ask three Avgi the same question and you'll get six different answers. We are a fractious lot, us Avgi, and that will be true until the stars burn out and the moons fall from the sky."`
+				goto humans
+
+			label believe
+				`	"Amusement. Neither. Either one would be too easy, and that of course can't be allowed, not in this universe. I expect your Republic would spend years and years debating whether to even acknowledge our existence, let alone take any kind of action."`
+			choice
+				`	"You don't give the Republic enough credit. Parliament can be decisive when it needs to be."`
+				`	"Parliament might be slow, but the Navy is full of good people who would help if they could."`
+				`	"That's a surprisingly accurate estimation of the Republic."`
+					goto accurate
+				`	"Maybe, but there are others that would try to exploit you. That's just human nature."`
+					goto exploit
+				`	"Well, the Republic isn't the only government in human space these days. The Free Worlds might try to help you."`
+					to display
+						has "event: war begins"
```

Comment:
It seems like most of this section's text does not display in the mission. (other than line 743) I am not well versed enough in mission construction to understand why though.